src/Castle.Facilities.AspNet.SystemWeb/Lifestyle/PerWebRequestAttribute.cs

```diff
@@ -12,21 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if FEATURE_SYSTEM_WEB
-namespace Castle.Core
+namespace Castle.Facilities.AspNet.SystemWeb.Lifestyle
 {
 	using System;
 
-	/// <summary>
-	///   Indicates that the target components wants a
-	///   per web request lifestyle.
-	/// </summary>
+	using Castle.Core;
+
 	[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
-	public sealed class PerWebRequestAttribute : LifestyleAttribute
+	public sealed class PerWebRequestAttribute : ScopedAttribute
```

Comment:
Very important. It fits in with the spirit of where the previous commits were going.